common/build/eslint-config-fluid/recommended.js

```diff
@@ -38,6 +38,11 @@ module.exports = {
             },
         ],
         "unicorn/empty-brace-spaces": "off",
+
+        // Rationale: Destructuring of `Array.entries()` in order to get the index variable results in a
+        //            significant performance regression [node 14 x64].
+        "unicorn/no-for-loop": "off",
```

Comment:
Interesting that it had to be disabled here and in eslint7.

Reply:
Hmmm, maybe the unicorn config should be extended before minimal?

Reply:
@tylerbutler - `@typescript-eslint/prefer-for-of` does what we want (already enabled).

Since you signed off, I assume you're okay w/me leaving the config inheritance issue for you and @Josmithr to reason though.  :-)